packages/zoe/src/contractSupport/topics.js

```diff
@@ -21,6 +21,16 @@ export const PublicTopicShape = M.splitRecord(
  * }} PublicTopic
  */
 
+/**
+ * A {PublicTopic} in which the `storagePath` is a string.
```

Comment:
Maybe add a why to one of the `Resolved*` types
```suggestion
 * A {PublicTopic} in which the `storagePath` is a string.
 * Useful when working with Vows and async-flow.
 ```

Reply:
good idea. we can use more "what this is for"